guide/14-deep-learning/edge-detection.ipynb

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
<p>You can optionally provide a <code>backbone</code> parameter, the default value for which is <code>vgg19</code>. In our implementation of Edge Detection models, ResNets can also be used apart from VGGNets as the backbones. </p><p> </p><p><strong>Q: Should this be "<em>ResNets can also be used as the backbone, rather than VGGNets</em>"?</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I mean to say that both ResNets and VGGNets can be used. The authors made it work with VGG, but additionally, we have provided the option of ResNets also.</p>

Reply:
<p>"You can optionally provide a <code>backbone</code> parameter, the default value for which is <code>vgg19</code>. In our implementation of Edge Detection models, <strong>ResNets can also be used as the backbone, in addition to VGGNets.</strong>"</p>